### ISSUE TITLE
Bot Deployed on Azure works from emulator but fails in web chat/ direct line

### ISSUE THREAD

**Issue by aparnaji** [microsoft/BotFramework-Composer#7716]
## Describe the bug

Bot Deployed on Azure works from emulator but fails in web chat/ direct line
## Version

v
![image](https://user-images.githubusercontent.com/26645750/117549365-93844900-b029-11eb-8596-ec6482a5ee57.png)

## Browser

- [ ] Chrome

- [ ] Edge

## OS

windows

## To Reproduce

1. Created a bot in composer
2. Worked locally, published to Azure app service
3. Created channel registration
4. "Test in Web Chat' fails in the Azure Channel
5. If tested from local emulator using http://xxx.azurewebsites.net/api/messages it works (provided appid and password while connecting)
6. Failed to load resource: the server responded with a status of 403 ()
Locally tried with the iframe and gave this error
code: "BadArgument"
message: "Site missing. Learn more about sites  https://docs.microsoft.com/azure/bot-service/bot-service-channel-connect-directline?view=azure-bot-service-4.0#add-new-site."

## Expected behavior
Should be able work with web chat

**Comment by aparnaji** [comment]
Adding further details , Message is "There was an error sending this message to your bot: HTTP status code GatewayTimeout" in the channel . My Bot uses both luis and qnamaker

**Comment by tdurnford** [comment]
@srinaath Can you take a look at this?

**Comment by aparnaji** [comment]
I fixed the issue,  I migrated to Composer 2.xxx (nightly build) assuming this problem will be fixed, but looks like the issue is wth Adaptive Card that I used in first dialogue. problems that I see are  1. used version 1.3, but I believe  it works with lower version 1.2 and also (2) invoking a function within the card ${greeting}, which is throwing error so replaced with hard coded text. "Have to see how I can send text to a adaptive card.
(3) replaced call to adaptive cards with  -${} instead of  -``` xxx`````,

I could not find anything related to this error in app insights, figured out by elimination method. Is there any way we can narrow down these issues
